crates/solidity/inputs/schema/grammar/05-expressions/03-numbers/productions.yml

```
@@ -10,11 +10,13 @@
             - reference: "DecimalLiteral"
         - optional:
             reference: "NumberUnit"
-    0.5.0:
-      sequence:
-        - reference: "DecimalLiteral"
-        - optional:
-            reference: "NumberUnit"
+    0.5.0: # hex literal no longer allows a number unit
```

nit suggestion, to make it easy to grep for rule usage/changes.
```suggestion
    0.5.0: # "HexLiteral" no longer allows a "NumberUnit"
```